tftrt/benchmarking-cpp/build-scripts/tf-profiler.patch

```diff
@@ -0,0 +1,22 @@
+--- /opt/tensorflow/tensorflow-source/tensorflow/core/profiler/rpc/client/BUILD	2022-06-24 20:49:49.656963813 +0000
++++ /opt/tensorflow/tensorflow-source/tensorflow/core/profiler/rpc/client/BUILD_PATCHED	2022-06-24 20:49:35.416963948 +0000
+@@ -25,6 +25,7 @@
+     visibility = [
+         "//tensorflow/compiler/xla/python:__pkg__",
+         "//tensorflow/python/profiler/internal:__pkg__",
++        "//tensorflow:internal",
+     ],
+     deps = [
+         ":profiler_client_for_pybind",
+@@ -67,7 +68,10 @@
+ cc_library(
+     name = "profiler_client",
+     hdrs = ["profiler_client.h"],
+-    visibility = ["//tensorflow/compiler/xla:__subpackages__"],
```

Comment:
@cliveverghese couldn't we just make `profiler_client.h` a public header ? I really don't see what headache it would create.
TF Profiler is very useful, this usecase might just need `ExportToTensorBoard` another project might need more. I can hardly see why we would restrict the visibility of the entire header. Maybe I'm missing something :)

Reply:
Hi @DEKHTIARJonathan,

We would like to keep the other functionality in the file, ie, `Monitor` and `Trace` as private in its current form because the functions/signatures could change. That was not the case with `ExportToTensorBoard`.